### PR TITLE
Fix orbitgt pointcloud position

### PR DESCRIPTION
**Pull request by lerzeel2**
Use more accurate transform to properly position orbitgt pointclouds in the model. This resolves elevation shifts that can be seen on pointcloud data that is far away from the imodel center.

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

**Review by RBBentley — APPROVED** [review]

File: core/frontend/src/tile/OrbitGtTileTree.ts
Comment (lerzeel2) [reply core/frontend/src/tile/OrbitGtTileTree.ts:385]: The distance check has been added as suggested.

File: core/frontend/src/tile/OrbitGtTileTree.ts
Comment (lerzeel2) [reply core/frontend/src/tile/OrbitGtTileTree.ts]: The comment has been updated as suggested.

**Comment by pmconne** [comment]
@lerzeel2 is this waiting for something before it can be merged?

**Pull request merged**
